libbeat/outputs/elasticsearch/client_proxy_test.go

```diff
@@ -221,17 +221,17 @@ type serverState struct {
 	serverURL string
 	proxyURL  string
 
-	_serverRequestCount atomic.Int // Requests directly to the server
-	_proxyRequestCount  atomic.Int // Requests via the proxy
+	_serverRequestCount atomic.Int64 // Requests directly to the server
+	_proxyRequestCount  atomic.Int64 // Requests via the proxy
```

Comment:
The original is 32 bits wide. Do we care? There is almost no-one in the world who that would affect, and it would have affected them anyway. Raising only for awareness.